Makefile

```diff
@@ -59,7 +59,7 @@ endef
 ######################
 
 NONFREE_REMOTE ?= git@iis-git.ee.ethz.ch:pulp-restricted/pulp-cluster-nonfree.git
-NONFREE_COMMIT ?= ad58d90
+NONFREE_COMMIT ?= 06cb2203
```

Comment:
Would you please also open a PR also on the nonfree if we need to update?

Reply:
Done. See: https://iis-git.ee.ethz.ch/pulp-restricted/pulp-cluster-nonfree/-/merge_requests/11